resources/slack-openapi-patched.json

```diff
@@ -100,8 +100,8 @@
         },
         "defs_ts": {
             "pattern": "^\\d{10}\\.\\d{6}$",
-            "title": "Timestamp in format 0123456789.012345",
```

Comment:
This "title" change is not in the patch, is that manual?

Reply:
Oh I don't know why this happens. I've changed the patch.